pkgs/development/libraries/xdg-desktop-portal/default.nix

```diff
@@ -142,6 +144,22 @@ stdenv.mkDerivation (finalAttrs: {
     export TEST_IN_CI=1
   '';
 
+  postFixup = let
+    documentFuse = "${placeholder "installedTests"}/libexec/installed-tests/xdg-desktop-portal/test-document-fuse.py";
+    testPortals = "${placeholder "installedTests"}/libexec/installed-tests/xdg-desktop-portal/test-portals";
+
+  in ''
+    if [ -x '${documentFuse}' ] ; then
+      wrapGApp '${documentFuse}'
+      wrapGApp '${testPortals}'
+      cp -r tests/share/applications $installedTests/share
```

Comment:
Why does not upstream install this?

Reply:
https://github.com/flatpak/xdg-desktop-portal/pull/1444

Reply:
Hmm now it simply doesn't work and doesn't detect the files even though the share is listed and contains thefiles

```
vm-test-run-xdg-desktop-portal> # xdg-desktop-portal-DEBUG: Setting XDG_DATA_DIRS to /nix/store/18anzwnlkpl348yg1j94hikywc39kaah-xdg-desktop-portal-1.18.4/share:/nix/store/99kdsvjp77yv0yxs9wzpn8x227j7d1zc-gsettings-desktop-schemas-46.0/share/gsettings-sche
mas/gsettings-desktop-schemas-46.0:/nix/store/nzll3s1gzpmqiij4vyanpnpybryx8i0f-gtk+3-3.24.43/share/gsettings-schemas/gtk+3-3.24.43:/nix/store/fm8lwmakyhk770r9vc6db4bf68mcm88g-pipewire-1.2.3/share/gsettings-schemas/pipewire-1.2.3:/nix/store/j6s741bjd5wl3pa5
lwq5j7lg0s83z0fg-xdg-desktop-portal-1.18.4-installedTests/share/installed-tests/xdg-desktop-portal/share:/root/.nix-profile/share:/nix/profile/share:/root/.local/state/nix/profile/share:/etc/profiles/per-user/root/share:/nix/var/nix/profiles/default/share:
/run/current-system/sw/share:/usr/local/share/:/usr/share/
```

Reply:
Had to install to `.../share/applications` now

Reply:
It's been merged